projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -323,32 +323,39 @@ class_exists( 'Jetpack' ) ||
 		Assets::enqueue_script( 'jetpack-social-editor' );
 
 		$jetpack_social_settings = new Automattic\Jetpack\Publicize\Jetpack_Social_Settings\Settings();
-		$initial_state           = $jetpack_social_settings->get_initial_state();
+		$social_state            = $jetpack_social_settings->get_initial_state();
+
+		$initial_state = array(
+			'adminUrl'                        => esc_url_raw( admin_url( 'admin.php?page=jetpack-social' ) ),
+			'sharesData'                      => $publicize->get_publicize_shares_info( Jetpack_Options::get_option( 'id' ) ),
+			'connectionRefreshPath'           => '/jetpack/v4/publicize/connection-test-results',
+			'resharePath'                     => '/jetpack/v4/publicize/{postId}',
+			'publicizeConnectionsUrl'         => esc_url_raw(
+				'https://jetpack.com/redirect/?source=jetpack-social-connections-block-editor&site='
+			),
+			'hasPaidPlan'                     => $publicize->has_paid_plan(),
+			'isEnhancedPublishingEnabled'     => $publicize->has_enhanced_publishing_feature(),
+			'isSocialImageGeneratorAvailable' => $initial_state['socialImageGeneratorSettings']['available'],
+			'isSocialImageGeneratorEnabled'   => $initial_state['socialImageGeneratorSettings']['enabled'],
+			'autoConversionSettings'          => $initial_state['autoConversionSettings'],
+			'useAdminUiV1'                    => $initial_state['useAdminUiV1'],
```

Comment:
This seems to be a mistake here, it should be `$social_state` instead of `$initial_state`

```suggestion
			'isSocialImageGeneratorAvailable' => $social_state['socialImageGeneratorSettings']['available'],
			'isSocialImageGeneratorEnabled'   => $social_state['socialImageGeneratorSettings']['enabled'],
			'autoConversionSettings'          => $social_state['autoConversionSettings'],
			'useAdminUiV1'                    => $social_state['useAdminUiV1'],
```